coreneuron/mechanism/mech/mod2c_core_thread.hpp

```diff
@@ -68,14 +68,14 @@ struct Item {
 
 using List = Item; /* list of mixed items */
 
-struct SparseObj {          /* all the state information */
-    Elm** rowst;            /* link to first element in row (solution order)*/
-    Elm** diag;             /* link to pivot element in row (solution order)*/
-    void* elmpool;          /* no interthread cache line sharing for elements */
-    unsigned neqn;          /* number of equations */
-    unsigned _cntml_padded; /* number of instances */
-    unsigned* varord;       /* row and column order for pivots */
-    double* rhs;            /* initially- right hand side        finally - answer */
+struct SparseObj: public MemoryManaged { /* all the state information */
```

Comment:
Some changes like this are part of an incomplete attempt to fix https://github.com/BlueBrain/CoreNeuron/issues/594.